Tests/TestUtilities.psm1

```diff
@@ -88,9 +88,8 @@ function Initialize-Test {
     # Module generation part needs to happen in full powershell
     Write-Verbose "Generating module"
     if((Get-Variable -Name PSEdition -ErrorAction Ignore) -and ('Core' -eq $PSEdition)) {
-        & "powershell.exe" -command "& {`$env:PSModulePath=`$env:PSModulePath_Backup;
+        & "$global:fullPowerShellPath\powershell.exe" -command "& {`$env:Path = '$global:fullPowerShellPath;$env:Path';`$env:PSModulePath=`$env:PSModulePath_Backup;
             Import-Module (Join-Path `"$PsSwaggerPath`" `"PSSwagger.psd1`") -Force;
```

Comment:
>Import-Module (Join-Path `"$PsSwaggerPath`" `"PSSwagger.psd1`") -Force; [](start = 12, length = 71)

Please add this line along with powershell.exe with full path in the new utility function I have added in my last PR.

Reply:
Working on it - see https://github.com/PowerShell/PSSwagger/pull/324#issuecomment-328243267